src/docs/pages/components/variants/textArea/Required.tsx

```diff
@@ -0,0 +1,25 @@
+import { FormEvent, useState } from 'react';
+import Form from 'library/components/Form';
+import TextArea from 'library/components/TextArea';
+import Example from 'docs/shared/components/Example';
+
+const RequiredTextArea = (): JSX.Element => {
+  const [didSubmit, setDidSubmit] = useState(false);
+
+  const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setDidSubmit(true);
+  };
+
+  return (
+    <Example>
+      <Form onSubmit={handleSubmit}>
+        <div className="bsds-form-control bsds-margin-top-remove">
```

Comment:
do we need bsds-margin-top-remove here? is it because it's nested in the form and others aren't?

Reply:
yes, the form adds a 2rem top-margin otherwise